Offseason2022/src/main/java/frc/robot/commands/IntakeRun.java

```diff
@@ -14,12 +14,12 @@
 public class IntakeRun extends CommandBase
 {
   private Intake m_intake;
-  private Mode   m_intakeDirection;
+  private Mode   m_mode;
 
   public void intakeRun(Intake intake, Mode intakeDirection)

```

Comment:
Looks great except for I don't think it compiles yet. For the constructor intakeRun, it has a parameter intakeDirection, but in the body of the method you've changed it (correctly) to be "mode".

Once it compiles with that change, let's have at least one other person review and get this merged.